webapp/channels/src/components/common/hooks/useGetTotalUsersNoBots.ts

```diff
@@ -5,15 +5,14 @@ import {useEffect, useState} from 'react';
 import {useDispatch} from 'react-redux';
 
 import {getFilteredUsersStats} from 'mattermost-redux/actions/users';
-import type {DispatchFunc} from 'mattermost-redux/types/actions';
 
 const useGetTotalUsersNoBots = (includeInactive = false): number => {
-    const dispatch = useDispatch<DispatchFunc>();
+    const dispatch = useDispatch();
     const [userCount, setUserCount] = useState<number>(0);
 
     const getTotalUsers = async () => {
         const {data} = await dispatch(getFilteredUsersStats({include_bots: false, include_deleted: includeInactive}, false));
-        setUserCount(data?.total_users_count);
+        setUserCount(data?.total_users_count ?? 0);
```

Comment:
Sometimes you just gotta fix stuff :P

Reply:
Yeah, I guess I didn't feel like messing up the type definitions here. In this case, it's because it already had `?.` which meant I couldn't just add `!.` to it